main.go

```diff
@@ -0,0 +1,663 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/agrison/go-tablib"
+	color "github.com/fatih/color"
+	"github.com/jinzhu/configor"
+	"github.com/jinzhu/gorm"
+	"github.com/k0kubun/pp"
+	"github.com/kamilsk/breaker"
+	"github.com/kamilsk/retry"
+	"github.com/kamilsk/retry/strategy"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/qor/admin"
+	"golang.org/x/net/proxy"
+	// "github.com/spf13/cobra"
+)
+
+/*
+Similar:
+- https://github.com/mesuutt/sherlock/blob/master/main.go
```

Comment:
Lol... I never agree with this source code is similar to that.

Reply:
do you have a telegram account for discussing ?

Reply:
Yes, I have an account for telegram: @tdh8316.